### DISCUSSION TITLE
epub with gt tables - forced single column output

### DISCUSSION THREAD

**Discussion by lauren-obrien** [quarto-dev/quarto-cli#8643] [Q&A]
### Description

Quarto 1.5.8, Windows 10, gt 0.10.1. Project type 'book'.

When rendering a `gt` table in epub format, all the table data is being coerced into a single column, so e.g. for the following code

````qmd
---
title: "table test"
format: epub
engine: knitr
---

```{r}
#| label: tbl-one
#| tbl-cap: "Test table"

library(dplyr)
library(tibble)
library(gt)

dat_one <-
  tibble::tribble(~One, ~Two,
          'A', 'Apple',
          'B', 'Banana',
          'C', 'Pear')

gt(dat_one)
``` 
````
I get 

![image](https://github.com/quarto-dev/quarto-cli/assets/46914665/52e439e7-ef3f-4f00-8b0f-5d44fa821f58)

Do I need to add something to the yaml, or are `gt` and ePub just not really ready to work together yet?

**Comment by mcanouil** [comment]
If you could edit and format your example to make it look like the one below, that would be great.

See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

![A placeholder image](https://placehold.co/600x400.png)

The end.
````
`````

---

I cannot reproduce:

<table>
<tr><th>Quarto document</th><th>HTML</th></tr>
<tr><td>

````qmd
---
title: Playground
format: epub
---

```{r}
#| label: tbl-one
#| tbl-cap: "Test table"
library(tibble)
library(gt)
gt(tibble::tribble(~One, ~Two, 'A', 'Apple', 'B', 'Banana', 'C', 'Pear'))
```

````

</td><td>

<img width="494" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/f4e5d508-ca70-45e2-99a8-a53790be3cb1">

</td></tr>
</table>

**Comment by lauren-obrien** [comment]
Edited. On further investigation though, this appears to be a problem specific to SumatraPDF software (which is a multiformat reader despite the name). Calibre E-reader displays the reprex correctly.
